### PR TITLE
fix(chat): export import integration with latest API (issue #640)

### PR DESCRIPTION
**Pull request by denys-kolomiitsev**
- #640

### CODE REVIEW

File: apps/chat/src/utils/app/data/prompt-service.ts [hunk 1]
```diff
@@ -71,3 +71,30 @@ export const getPreparedPrompts = ({
...
+export const getImportPreparedPrompts = ({
+  prompts,
+  folders,
+}: {
+  prompts: Prompt[];
+  folders: FolderInterface[];
+}) =>
+  prompts.map((prompt) => {
+    const { path } = getPathToFolderById(folders, prompt.folderId, true);
+    const newName = prompt.name.replace(notAllowedSymbolsRegex, '');
```
Comment (Alexander-Kezik) [inline apps/chat/src/utils/app/data/prompt-service.ts]:
```suggestion 
    const newName = prepareEntityName(prompt.name);
```
names length can't be more than 160 symbols now.

File: apps/chat/src/utils/app/data/storages/api/conversation-api-storage.ts [hunk 2]
```diff
@@ -113,3 +113,38 @@ export const getPreparedConversations = ({
...
+  conversationsFolders: FolderInterface[];
+}) =>
+  conversations.map((conv) => {
+    const { path } = getPathToFolderById(
+      conversationsFolders,
+      conv.folderId,
+      true,
+    );
+
+    const newName = conv.name.replace(notAllowedSymbolsRegex, '');
```
Comment (Alexander-Kezik) [inline apps/chat/src/utils/app/data/storages/api/conversation-api-storage.ts]:
```suggestion
    const newName = prepareEntityName(conv.name);
```
names length can't be more than 160 symbols now.

File: apps/chat/src/utils/app/data/prompt-service.ts (hunk 1 above)
Comment (denys-kolomiitsev) [reply apps/chat/src/utils/app/data/prompt-service.ts]: Thank you for the suggestion!
New prepareEntityName function used in all needed places.

File: apps/chat/src/utils/app/data/storages/api/conversation-api-storage.ts (hunk 2 above)
Comment (denys-kolomiitsev) [reply apps/chat/src/utils/app/data/storages/api/conversation-api-storage.ts]: Thank you for the suggestion!
New prepareEntityName function used in all needed places.

**Review by Alexander-Kezik — APPROVED** [review]

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-752.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-752.nightly-test.deltixhub.io
>E2E tests status: success

**Pull request merged**